docs/gc.md

```diff
@@ -83,17 +76,17 @@ The *nursery* area is subdivided into two further areas, the *allocate* space an
+The relative sizes of the *allocate* and *survivor* spaces are dynamically adjusted by a technique called *tilting*. When the *nursery* area is first created, it is evenly divided between the *allocate* and *survivor* spaces. If, after a GC *scavenge* operation runs, the amount of space required for the *survivor* area is comparatively small, the boundary between the two spaces is adjusted by *tilting*. For example, if the *survivor* space requires only 10% of the *nursery* area, the *tilt ratio* is adjusted to give 90% of the *nursery* area to the *allocate* space. With more space available for new objects, a global GC cycle can be delayed.
 
-The *tenure age* of an object is determined by the VM and reflects the number of times that an object has been copied between the *allocate* space and the *survivor* space. The age is in the range 1 - 14 and is adjusted dynamically by the VM depending on the overall amount of space that is used in the *nursery area*. For example, if an object has a tenure age of 5, it has been copied backwards and forwards between *allocate* and *survivor* spaces 5 times. If the VM sets a *tenure age* of 5 based on the percentage of space remaining in the *nursery* area, the next scavenge moves the object from the *nursery* to the *tenure* area. You can set an initial *tenure age* with the [`-Xgc:scvTenureAge`](xgc.md#scvtenureage) option. You can also prevent the VM dynamically adjusting the *tenure age* by setting the [`Xgc:scvNoAdaptiveTenure`](xgc.md#scvnoadaptivetenure) option so that the intial age is maintained throughout the run time of the VM.
+The *tenure age* of an object is determined by the VM and reflects the number of times that an object has been copied between the *allocate* space and the *survivor* space. The age is in the range 1 - 14 and is adjusted dynamically by the VM depending on the overall amount of space that is used in the *nursery area*. For example, if an object has a tenure age of `5`, it has been copied backwards and forwards between *allocate* and *survivor* spaces five times. If the VM sets a *tenure age* of `5` based on the percentage of space remaining in the *nursery* area, the next *scavenge* moves the object from the *nursery* to the *tenure* area. You can set an initial *tenure age* with the [`-Xgc:scvTenureAge`](xgc.md#scvtenureage) option. You can also prevent the VM dynamically adjusting the *tenure age* by setting the [`Xgc:scvNoAdaptiveTenure`](xgc.md#scvnoadaptivetenure) option so that the initial age is maintained throughout the run time of the VM.
```

Comment:
Actual heuristics for determining tenure age are rather complicated, but it would be more correct to say:

'... adjusted dynamically by the VM depending on the object survival rate at particular ages'

or more precisely (but probably overkill):

'... adjusted dynamically by the VM based on changes in object survival rates from one age to the next one'
(in math terms we are looking into first derivative).

Reply:
Went with:

"The age is in the range 1 - 14 and is adjusted dynamically by the VM depending on the object survival rate at particular ages."